cuda_bindings/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 recursive-include cuda/ *.pyx *.pxd
 # at least with setuptools 75.0.0 this folder was added erroneously
 # to the payload, causing file copying to the build environment failed
-exclude cuda/bindings
+exclude cuda/bindings cuda?bindings
```

Comment:
To be clear this is NOT a bug fix (the one-liner change below is), just a really unpleasant workaround for presumably two (2) potentially correlated setuptools bugs. Many engineering hours are wasted only to find this WAR in order to accommodate for both Linux and Windows. Yes, we need both; can't just pick either of them. 

What happened is (paraphrasing @vzhurba01's comment from the internal discussion): We need to fix `pyproject.toml` to find all extension modules starting with `cuda` (which was missed in the hot fix 12.6.2/11.8.5), so that all modules' Cython sources can be included in the package. 

However, once we do this we see a very strange error on both platforms at build time
```shell
error: can't copy 'cuda/bindings': doesn't exist or not a regular file
```
which suggests that setuptools tries to copy this folder as a regular file. This is likely the first bug.

What's happening here is that as part of `egg_info` source file discovery, it fetches files directly from `build_ext`. Files found use a mix of Linux and Windows path separators (`/` and `\\`, respectively). One of the paths found is `cuda/bindings` (which should not have happened in the first place). 

To work it around, we rely on `MANIFEST.in` letting us specify which path/file to exclude, and so we added `exclude cuda/bindings` when we worked on the layout change #137. Since [setuptools' docs](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html) clearly stated that 
> File & directory names in `MANIFEST.in` should be `/`-separated; setuptools will automatically convert the slashes to the local platform’s appropriate directory separator.

this should have been a cross-platform WAR.

The next bug that bites us is that `egg_info` tries to be "smart" by converting all Linux separators to Windows, therefore our `exclude` only applies to `cuda\\bindings` and thus failing the Windows build later on. This is the second bug.

Reply:
@thomasjpfan in case you're still interested in knowing what happened 🙂